lib/inih/ini.c

```diff
@@ -194,6 +206,7 @@ int ini_parse_stream(ini_reader reader, void* stream, ini_handler handler,
 #endif
                 value = lskip(value);
                 rstrip(value);
+                unquote(value);
```

Comment:
Well, this still won't work because you are discarding return value from the `unquote` function.

Reply:
Pushed a commit to correct this; thanks.